package.json

```diff
@@ -39,13 +39,13 @@
   },
   "dependencies": {
     "@nomiclabs/hardhat-ethers": "^2.0.6",
+    "@openzeppelin/hardhat-upgrades": "^1.19.0",
```

Comment:
Also bump to [`1.19.1`](https://github.com/OpenZeppelin/openzeppelin-upgrades/releases/tag/%40openzeppelin%2Fhardhat-upgrades%401.19.1)?